packages/core/src/common/classes.ts

```diff
@@ -193,8 +193,6 @@ export const SPINNER = `${NS}-spinner`;
 export const SPINNER_HEAD = `${SPINNER}-head`;
 export const SPINNER_NO_SPIN = `${NS}-no-spin`;
```

Comment:
while we're at it, thoughts on a better name for this? something more generic like `.pt-static` maybe? or maybe indeterminate vs determinate loading?